main.ts

```diff
@@ -28,15 +44,19 @@ export default class RelativeLineNumbers extends Plugin {
     if (this.isLegacy()) {
       this.legacyEnable();
     } else {
-      // @ts-ignore
-      this.registerEditorExtension(lineNumbersRelative());
+      this.editorExtension.length = 0;

```

Comment:
this follows code style from these guidelines https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Change+or+reconfigure+editor+extensions